### PR TITLE
Controllers test

### PR DESCRIPTION
**Pull request by kensteph**
In this PR I:
- Created a Request spec file for all controllers.
- Checked response for action:
  - If the response status was correct.
  - If a correct template was rendered.
  - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+      expect(response).to have_http_status(200)
+    end
+
+    it 'index template was rendered' do
+      get user_posts_path(user)
+
+      expect(response).to render_template(:index)
+    end
+  end
```
Comment (anagudelogu) [inline spec/requests/posts_spec.rb]: - [ ] You did a great job working on this milestone! 💯 However, please make sure to include a test for the `response.body` containing the correct placeholder text. According to the requirements:

image

```rb
# Example
it 'response body includes the correct content' do
  get user_posts_path(user)
  expect(response.body).to match(/posts index/)
end
```


Please consider including this! 🤓 🔧

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+      expect(response.status).to eq(200)
+    end
+
+    it 'render template ' do
+      get user_post_path(user, post)
+
+      expect(response).to render_template(:show)
+    end
+  end
```
Comment (anagudelogu) [inline spec/requests/posts_spec.rb]: - [ ] Here happens the same, please make sure to include a test for the `response.body` containing the correct placeholder text. 🤓 🔧

File: spec/requests/users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      get users_path
+      expect(response).to have_http_status(200)
+    end
+
+    it 'index template was rendered' do
+      get users_path
+
+      expect(response).to render_template(:index)
+    end
+  end
```
Comment (anagudelogu) [inline spec/requests/users_spec.rb]: - [ ] Here happens the same, please make sure to include a test for the `response.body` containing the correct placeholder text. 🤓 🔧

File: spec/requests/users_spec.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+      expect(response.status).to eq(200)
+    end
+
+    it 'render show' do
+      get user_path(user.id)
+
+      expect(response).to render_template(:show)
+    end
+  end
```
Comment (anagudelogu) [inline spec/requests/users_spec.rb]: - [ ] Here happens the same, please make sure to include a test for the `response.body` containing the correct placeholder text. 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @kensteph, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* All tests passed 
* Well written PR 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**